lib/st25r3911b/st25r3911b_reg.h

```diff
@@ -532,8 +532,9 @@ extern "C" {
 #define ST25R3911B_REG_REGULATOR_CTRL_MPSV_VSP_D (2 << 1)
 #define ST25R3911B_REG_REGULATOR_CTRL_MPSV_VSP_RF (3 << 1)
 #define ST25R3911B_REG_REGULATOR_CTRL_REGE0 BIT(3)
-#define ST25R3911B_REG_REGULATOR_CTRL_REGE2 BIT(4)
-#define ST25R3911B_REG_REGULATOR_CTRL_REGE3 BIT(5)
+#define ST25R3911B_REG_REGULATOR_CTRL_REGE1 BIT(4)
+#define ST25R3911B_REG_REGULATOR_CTRL_REGE2 BIT(5)
+#define ST25R3911B_REG_REGULATOR_CTRL_REGE3 BIT(6)
 #define ST25R3911B_REG_REGULATOR_CTRL_REG_S BIT(6)
```

Comment:
@Andrew55529 thanks for preparing PR.
Isn't `reg_s` the 7th bit?
```suggestion
#define ST25R3911B_REG_REGULATOR_CTRL_REG_S BIT(7)
```